eahub/templates/eahub/group.html

```diff
@@ -80,7 +80,7 @@ <h1 class="word-break">{{ group.name }}</h1>
         {% if group.city_or_town %}<li class="list-group-item"><b>City/Town: </b>{{ group.city_or_town }}</li>{% endif %}
         {% if group.country %}<li class="list-group-item"><b>Country: </b>{{ group.country }}</li>{% endif %}
         {% if group.public_organisers %}
-        <li class="list-group-item"><b>Organisers:</b>
+        <li class="list-group-item"><b>Organisers (Member):</b>
```

Comment:
I think it would be better to add the non-members to the Organisers list (just without links), instead of having an extra bullet point